src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -343,22 +390,31 @@ public void Execute_WithErrors_CreatesSnapshotAndA11yTestFile()
             expectedResults.ErrorCount = 75;
             InitResultsCallback(expectedResults);
 
-            var expectedPath = "Test.file";
 
             _actionsMock.Setup(x => x.CaptureScreenshot(It.IsAny<Guid>()));
-            _actionsMock.Setup(x => x.SaveA11yTestFile(expectedPath, It.IsAny<A11yElement>(), It.IsAny<Guid>()));
-
+            _actionsMock.Setup(x => x.SaveA11yTestFile(It.IsAny<string>(), It.IsAny<A11yElement>(), It.IsAny<Guid>()));
+            _resultsAssemblerMock.Setup(x => x.AssembleScanResultsFromElement(It.IsAny<A11yElement>())).Returns(() => new ScanResults() { ErrorCount = 1});
             _outputFileHelperMock.Setup(m => m.EnsureOutputDirectoryExists());
-            _outputFileHelperMock.Setup(x => x.GetNewA11yTestFilePath()).Returns(expectedPath);
+            _outputFileHelperMock.Setup(x => x.GetNewA11yTestFilePath(It.IsAny<Func<string, string>>())).Returns((Func<string, string> decorator) => decorator("Test.File"));
 
             var config = Config.Builder
                 .ForProcessId(-1)
                 .WithOutputFileFormat(OutputFileFormat.A11yTest)
                 .Build();
 
             var actualResults = SnapshotCommand.Execute(config, _scanToolsMock.Object);
-            Assert.AreEqual(75, actualResults.ErrorCount);
-            Assert.AreEqual(expectedPath, actualResults.OutputFile.A11yTest);
+            var expectedPaths = new string[]
+            {
+                "Test.File_1_of_3",
```

Comment:
This output seems wrong. I'd expect `Test_1_of_3.File`

Reply:
Yeah that is just the test mock being wired in such a way that "Test.File" would be the regular file name.